update docs for `mev-boost-rs` (#142)
ralexstokes/mev-rs: book/mev-boost-rs.md

# `mev-boost-rs`

This repository provides a "boost" implementation that serves the [`builder-specs` APIs]
connecting validators to an external block builder network.

> Warning: this software is extremely experimental. You should not use in any production capacity.

## Installation

### Build from source

> The best way right now to use this right now is to build from source.

Author: ```suggestion
The best way to use this right now is to build from source.
```

## Run `mev-boost-rs`

> You'll need to put some configuration into a config file like `example.config.toml` found

Author: ```suggestion
You'll need to construct a configuration file like the `example.config.toml` found
```
